integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/noarkexchange/receive/InternalQueue.java

```diff
@@ -166,10 +164,30 @@ public void dlqListener(byte[] message, Session session) {
         } catch (Exception e) {
         }
 
+        try {
+            ConversationResource cr = unmarshalNextMoveMessage(message);
+            errorMsg = "Failed to send message. Moved to DLQ";
+            Audit.error(errorMsg, ConversationResourceMarkers.markerFrom(cr));
+            conversationId = cr.getConversationId();
+        } catch (Exception e) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1166)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either remove or fill this block of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS00108)